docs/updates/migrate-v4.md

```diff
@@ -121,3 +121,17 @@ Non-public `DuplexCollection#byGlobSource` API has been removed.
 - **Dependencies**: The `@ui5/builder` is now an optional dependency to the `@ui5/project`
 
 Consumers of the Node.js API that make use of the `ProjectGraph#build` method need to declare a dependency to `@ui5/builder` in their respective `package.json`. The package manager should ensure that the version fulfills the range specified in the `package.json` of `@ui5/project`.
+
+## Troubleshooting
+
+### App Stops Working With UI5 1.71
+
+This might caused by a mismatch of your project's manifest version and the UI5 version.
+
+UI5 Tooling v4 enhances the manifest.json with information about the `supportedLocales` (also see [Builder: Generation of Supported Locales](../pages/Builder.md#generation-of-supported-locales)). But in UI5 1.71 this configuration is not supported and leads to an ambiguous error message `TypeError: invalid input` and a failure to process the manifest file.
+
+UI5 Tooling uses the manifest's `_version` property to decide whether the `supportedLocales` can be generated or not. For UI5 1.71, only versions up to `1.17.0` are supported. See [UI5 Demo Kit: Descriptor for Applications, Components, and Libraries (manifest.json)](https://ui5.sap.com/#/topic/be0cf40f61184b358b5faedaec98b2da) for a mapping per manifest version.
```

Comment:
maybe better use the corresponding sdk.openui5.org link in this open source text

Reply:
@silkearians just made me ware of https://github.com/SAP/ui5-manifest/blob/main/mapping.json (a machine readable version of the mapping)

Reply:
Done